application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/BundleGroupPage.js

```diff
@@ -88,6 +89,16 @@ const BundleGroupPage = () => {
     })()
   }, [bundleGroupVersionId])
 
+  // checks the contact-us url for discover
+  const checkContactUsUrl = (string) => {
```

Comment:
Minor, this can be simplified.
const useContactUsModal = (url) => {
   return url && url.includes("discover.entando.com")
}

Reply:
Updated to the simplified version as above. 
Thank you